lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -1517,6 +1517,15 @@ def update():
     UUID: PGUuid,
 }
 
+
+class MACADDR8(MACADDR):
+    __visit_name__ = "MACADDR8"
+
+
+class TIMESTAMPTZ(TIMESTAMP):
```

Comment:
TIMESTAMPTZ is just a PG-specific abbreviation for TIMESTAMP WITH TIMEZONE and is unnecessary, so remove this

Reply:
Please, review the changes.